changelog/v4.md

```diff
@@ -0,0 +1,112 @@
+## 4.0.0 - Beta (WIP)
+
+Create Flex Plugin 4.0 uses the [Twilio Flex Plugins API](https://www.twilio.com/docs/flex/plugins/api) to manage your plugins, create versions, and 
+
+It also completely removes the dependency on `react-scripts` and instead uses native Webpack to build and serve your plugins locally.
```

Comment:
"also" seems unnecessary here, especially since you use it again in the next sentence